a pull request

linhphi9x94 opened the pull request.

andrewcox committed:
[maven-release-plugin] prepare for next development iteration

andrewcox committed:
Move to latest snapshot nifty/swift dependencies

andrewcox committed:
Fix recursive type stack double-push, double-pop bug

tageorgiou committed:
Add in config option to set traffic class (network QOS) used by the server.

facebook-github-bot-1 commented:
Thank you for your pull request and welcome to our community. We require contributors to sign our Contributor License Agreement, and we don't seem to have you on file. In order for us to review and merge your code, please sign up at https://code.facebook.com/cla - and if you have received this in error or have any questions, please drop us a line at cla@fb.com. Thanks!

andrewcox committed:
Allow recursive union fields which are not explicitly marked 'optional'

andrewcox committed:
Move union field requiredness checking earlier

We only respect metadata errors that are in place before the metadata builder
build() method is called, so adding this error was useless as is, since
it
happened inside the invocation of build().

andrewcox committed:
Ensure no metadata errors or warnings are added after the point they are thrown

andrewcox committed:
Comment on why 'optional' fields aren't allowed in unions

andrewcox commented:
@linhphi9x94 this looks like a mistake: 0.19.x is for hotfixes to the 0.19.x releases, not all commits from master should go there. Let me know if there is something specific you wanted.
